src/main/java/org/trigon/hardware/phoenix6/Phoenix6Inputs.java

```diff
@@ -30,15 +35,31 @@ public void toLog(LogTable table) {
         latestTable = table;
     }
 
+    /**
+     * Registers a threaded signal.
+     * Threaded signals use threading to process certain signals separately, keeping them running smoothly in the background.
+     * This avoids slowing down the main program, unlike simpler signals that run directly in it.
+     * This is also used for signals that need to be updated at a high frequency like for odometry.
```

Comment:
you're focusing too much on the unrelated stuff 
the main reason we do this is for high freq signals
keep this jdoc consistent across all threaded stuff